DigitalLearningSolutions.Web/Styles/index.scss

```diff
@@ -131,7 +131,7 @@ h1#page-heading {
 
 .nhsuk-summary-list__value {
   @include govuk-media-query($until: tablet) {
-    word-break: break-word;
+    word-break: break-all;

```

Comment:
Ah - forgot to change here. Also, does the hack work ok on other main browsers?

Reply:
Yes, this adds an extra declaration for IE only, other browsers only see break-word. IE gets both and ignores the one it doesn't support.

Reply:
There are too many instances of this to add a css class to each of them so I've defined it using a mixin instead